slang-antlr/src/main/antlr4/com/sonarsource/slang/parser/SLang.g4

```diff
@@ -215,7 +216,24 @@ nativeBlock
   ;
 
 returnExpression
-  :  RETURN statement
+  :  RETURN statement?
+  ;
+
+jumpStatement
```

Comment:
We don't need this middle rule, just put directly break and continue to `atomicExpression`.
We would need it, if it would be used somewhere else in the grammar